tensorflow_datasets/image_classification/deep_weeds.py

```diff
@@ -21,7 +21,7 @@
 import tensorflow.compat.v2 as tf
 import tensorflow_datasets.public_api as tfds
 
-_URL = "https://nextcloud.qriscloud.org.au/index.php/s/a3KxPawpqkiorST/download"
+_URL = "https://docs.google.com/uc?export=download&id=1xnK3B6K6KekDI55vwJ0vnc2IGoDga9cj"
```

Comment:
@PrattJena Have you tried generating the dataset locally using this code?
The link is to the page itself rather than the file which causes the `download_manager` to download the HTML webpage code instead of `images.zip`

Reply:
I am seeing some examples given in the checksums which uses google drive link and i see them using the same link as i have provided. [Here](https://drive.google.com/uc?export=download&id=1NO1NKQSpyq3DMLEwiqA-BHIqXli8vtIL) is an example in accentdb.txt

Reply:
@NikhilBartwal I am trying to follow to [these ](https://github.com/AlexOlsen/DeepWeeds/commit/da084f62bcd1a2b0afeb8b81f1a27be3186391a1#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)changes

Reply:
I think there was some understanding, I was referring to the link being incorrect not the method. The URL that you have used in the code is not the `google drive` url. 
If you want to download a file from Drive as in the case of `accents_db` dataset, use [this](https://drive.google.com/u/0/uc?id=1xnK3B6K6KekDI55vwJ0vnc2IGoDga9cj&export=download) instead